.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
We should just add .idea to the .gitignore file at the root of the repo

Reply:
sounds good, just did that, thanks